pom.xml

```diff
@@ -225,7 +225,13 @@
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
-                <version>0.8.1</version>
```

Comment:
Version information is removed. Is that OK?

Reply:
Yes, I think so.  It works at least. 
Version information is given in the dependencies section already. 

To be honest, I am not sure which of below is correct way.
  1. write version for both dependency and plugin.
  1. write version only in dependency and the plugin section is for configuring the module only.  

2. looks more reasonable and the sample I saw does not have version info in the "plugin" section. 
That is why I think I should skip the version info here.

Reply:
I see, thanks. I think No.2 looks better.